src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/stores/DatagridStore.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import {action, autorun, observable} from 'mobx';
+import Requester from '../../../services/Requester';
+import metadataStore from './MetadataStore';
+
+export default class DatagridStore {
```

Comment:
`ResourceStore` ???

Reply:
But should it still be placed in the same folder? And when we are talking about that, should we still introduce some kind of `ResourceRequester`? Or do we wait with that until it becomes necessary? Might be already the case in the next PR, when try to delete elments from the datagrid.

Reply:
When thinking about it... This store is already containing datagrid specific information, like the page. So should we simply rename it, and leave it where it is? And other components are not using it?

Reply:
keep `DatagridStore` for now and maybe move some parts into a `ResourceRequester `